tests/pipeline/test_mida_trainer.py

```diff
@@ -0,0 +1,60 @@
+import numpy as np
+import pytest
+from numpy import testing
+from sklearn.datasets import make_blobs
+from sklearn.linear_model import LogisticRegression
+from sklearn.preprocessing import MinMaxScaler
+
+from kale.pipeline.mida_trainer import MIDATrainer
+
+N_SAMPLES = 200
```

Comment:
Is the value of 200 needed? Shall we reduce it, even though it might not save many resources?